### PR TITLE
Fixed instance variable @essage typo

### PR DESCRIPTION
**Pull request by bradleyd**
Small typo fix.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -24,3 +24,4 @@
 /.rvmrc
 /.yardoc
 /doc
+tags
```
Comment (benlangfeld) [inline .gitignore]: What's this about?

**Comment by benlangfeld** [comment]
@polysics Can you review this? I'm not sure why this wasn't causing a spec failure. It seems pretty critical to me.

File: .gitignore (hunk 1 above)
Comment (bklang) [inline .gitignore]: I suspect this is for ctags output, which is used for (among other things) method/class search within Vim. If that is indeed the intent, then I'd be fine with leaving this .gitignored, though we should probably convert it to `/tags` so it doesn't accidentally match anything deeper in the tree.

Comment (benlangfeld) [inline .gitignore]: I'd suggest that perhaps editor specific things like this should go in a global gitignore or `.git/info/exclude`. I don't think vim is really a voicemail concern, it's one's environment's concern.

Comment (bradleyd) [inline .gitignore]: yeah, tags is for exuberant-ctags.  It really helps when traversing unfamiliar code.  I actually did not mean to leave it in there.

Comment (benlangfeld) [inline .gitignore]: I'd personally prefer if you could rebase it out :)

Comment (bradleyd) [inline .gitignore]: sure, what would be the best way to go about this.  submit another pull request?
Thanks,
Brad

Comment (benlangfeld) [inline .gitignore]: You can just modify your commit and force push :)

Comment (bradleyd) [inline .gitignore]: should be void of tags ;)

**Comment by bradleyd** [comment]
it looks like load_message never gets called from run, do you want another pull request to fix this?

**Comment by JustinAiken** [comment]
Haha, that is bonkers.  Merged, I'm fixing making sure `#load_message` gets called now.
